distribution/src/main/release/samples/pom.xml

```diff
@@ -240,108 +289,8 @@
                 </pluginRepository>
             </pluginRepositories>
         </profile>
-        <profile>
-            <id>java9-plus</id>
-            <activation>
-                <jdk>[9,)</jdk>
-            </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>jakarta.xml.bind</groupId>
-                    <artifactId>jakarta.xml.bind-api</artifactId>
-                </dependency>
-                <dependency>
-                    <groupId>jakarta.annotation</groupId>
-                    <artifactId>jakarta.annotation-api</artifactId>
-                </dependency>
-                <dependency>
-                    <groupId>jakarta.xml.ws</groupId>
-                    <artifactId>jakarta.xml.ws-api</artifactId>
-                </dependency>
-                <dependency>
-                    <groupId>jakarta.activation</groupId>
-                    <artifactId>jakarta.activation-api</artifactId>
-                    <version>2.1.0</version>
-                </dependency>
-                <dependency>
-                    <groupId>jakarta.jws</groupId>
-                    <artifactId>jakarta.jws-api</artifactId>
-                </dependency>
-                <dependency>
-                    <groupId>com.sun.xml.messaging.saaj</groupId>
-                    <artifactId>saaj-impl</artifactId>
-                    <version>2.0.1</version>
-                    <scope>runtime</scope>
-                </dependency>
-                <dependency>
-                    <groupId>org.glassfish.jaxb</groupId>
-                    <artifactId>jaxb-runtime</artifactId>
-                </dependency>
-                <dependency>
-                    <groupId>org.glassfish.jaxb</groupId>
-                    <artifactId>jaxb-xjc</artifactId>
-                </dependency>
-                <dependency>
-                    <groupId>org.glassfish.corba</groupId>
-                    <artifactId>glassfish-corba-orb</artifactId>
-                    <version>4.2.2</version>
-                </dependency>
-            </dependencies>
-        </profile>
-        
-        <!-- 
-            TODO: Please remove, temporary repository settings in order to enable snapshots for
-            Spring Boot 3.0.0
-        -->
-        <profile>
-            <id>jakarta</id>
```

Comment:
@jimma The introduction of the `jakarta` profile is on purpose since we depend on milestone releases (and before - even on snapshots). In general, we should not have these repositories and once we have the Spring train releases (mid-November by plan), the `jakarta` profile could be phased away from main tree and samples. Hope it make sense.

Reply:
Thanks for the update @reta . I added back this profile with a better profile name `spring`. The https://issues.apache.org/jira/browse/CXF-8788 is created to remove this profile after the final release comes out in the coming weeks.

Reply:
@jimma the `jakarta` name is being used in Jenkins builds and the same profile is also present in [pom.xml](https://github.com/apache/cxf/blob/main/pom.xml#L426). Could you please rename it back to `jakarta` so we don't need to change jobs configurations? (fe we run `-Peverything,jakarta`)

Reply:
@reta Thanks for review. Changed back . I saw this profile is set to `activeByDefault`. Do we still need to active this profile with -P flag ?

Reply:
I think when profiles are used explicitly (like fe `-P`), the `activeByDefault` is not respected (since it is not default anymore). If you are curious, it is simple to check:

```
$ mvn help:active-profiles
```

vs explicit profiles with `-P` (this command fails for me, the `jakarta` profile is not picked up anymore)

```
$ mvn help:active-profiles -Peverything
```

Reply:
@reta I didn't know this and learned new thing.  Thanks.